security/mk-cacerts.sh

```diff
@@ -117,9 +118,17 @@ for FILE in certs/*.crt; do
     fi
 done
 
-if [ "$NO_KEYSTORE" = false ] ; then
+if [ "$NO_KEYSTORE" = false ] ; then # for jdk8, jdk11
     num_certs=$("$KEYTOOL" -v -list -storepass changeit -keystore cacerts | grep -c "Alias name:")
 else
     num_certs=$(find certs/* | wc -l)
 fi
 echo "Number of certs processed: $num_certs"
+
+# post verification: (nr.(mk-ca-bundle.pl) - nr.(already imported file) == (nr. (current files in certs folder)) || (nr. (alias in cacerts))
+certNum="$((certNum-alreadyExistsCounter))"
+if [ "$certNum" != "$num_certs" ]; then
```

Comment:
https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk18u/job/jdk18u-linux-x64-temurin_SmokeTests/

Reply:
ok, i see.
a new test need to perform after the compliation is done. 
something in https://github.com/adoptium/temurin-build/tree/master/test/functional/buildAndPackage

Reply:
Yes, i'm happy for this PR to do the check it is, but for #3034 we need the SmokeTest change, the reason being because the output from this script:
- <jdk17 : keystore
- >=jdk17 : src file certs
Both get fed into the OpenJDK build that then import either into the image/jdk/lib/security/cacerts keystore, there is always the possibility that build fails or is incorrect, so we need to ensure it is good. One other issue that concerns me is if say a bug or network glitch occurs in [mk-ca-bundle.pl](https://github.com/adoptium/temurin-build/blob/master/security/mk-ca-bundle.pl) Perl script and say it returns 0 certs, how do we catch that? Should we hard code the expected number (ie.135) as it very rarely changes, or do we have a test that actually checks the expected list like OpenJDK do: https://github.com/adoptium/jdk17u/blob/5de8b78504df916d88c9e01941c4e27bfe49e949/test/jdk/sun/security/lib/cacerts/VerifyCACerts.java#L50